invoice/src/test/java/org/killbill/billing/invoice/usage/TestContiguousIntervalConsumableInArrear.java

```diff
@@ -321,11 +318,8 @@ public boolean apply(final InvoiceItem input) {
         assertTrue(result.get(0).getStartDate().compareTo(startDate) == 0);
         assertTrue(result.get(0).getEndDate().compareTo(firstBCDDate) == 0);
 
-        // check item detail
-        List<UsageInArrearDetail> itemDetails = objectMapper.readValue(result.get(0).getItemDetails(), new TypeReference<List<UsageInArrearDetail>>() {});
-
-        assertEquals(itemDetails.size(), 1);
-        assertEquals(itemDetails.get(0).getAmount().compareTo(result.get(0).getAmount()),0);
+        // Because the existing items did not have any json (item details), the new one also don't have the details.
```

Comment:
See comment above, behavior seems a bit strange to me.

Reply:
The trick is the following: Because we don't have previous details, we can't compute exact new details, we only know the total amount that was already invoiced but not the per-unit amount, so we end up with something like:

Billed amount $4
```
{
  "tierDetails": [
    {
      "tier": 1,
      "tierUnit": "unit",
      "tierPrice": 1,
      "tierBlockSize": 100,
      "quantity": 5,
      "amount": 5
    }
  ],
  "amount": 5
}
```

Reply:
Pushed the change in 669bd343f132e40cf478347507a98da7736d54b6 , we can revisit if necessary